### PR TITLE
Add a new method to benchmarks: DoubleEnsemble

### PR DESCRIPTION
**Pull request by meng-ustc**
## How Has This Been Tested?
- [ ] Pass the test by running: `pytest qlib/tests/test_all_pipeline.py` under upper directory of `qlib`.
- [ ] If you are adding a new feature, test on your own test scripts.

## Screenshots of Test Results (if appropriate):
1. Pipeline test:
2. Your own tests:

## Types of changes

- [ ] Fix bugs
- [ ] Add new feature
- [ ] Update documentation

### CODE REVIEW

**Review by Derek-Wds — APPROVED** [review]
LGTM! :smile: 

Please sign the Contributor License Agreement.
Screenshot from 2021-02-23 17-04-39

File: qlib/contrib/model/double_ensemble.py [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+        bins_sr=10,
+        bins_fs=5,
+        decay=None,
+        sample_ratios=None,
+        sub_weights=None,
+        epochs=100,
+        **kwargs
+    ):
+        self.base = base  # "gbm" or "mlp", specifically, we use lgbm for "gbm"
+        self.k = k
```
Comment (you-n-g) [inline qlib/contrib/model/double_ensemble.py:35]: Please give more details about the parameters.  
Give it a full name at least

File: README.md [hunk 2]
```diff
@@ -232,6 +232,7 @@ Here is a list of models built on `Qlib`.
 - [SFM based on pytorch (Liheng Zhang, et al. 2017)](qlib/contrib/model/pytorch_sfm.py)
 - [TFT based on tensorflow (Bryan Lim, et al. 2019)](examples/benchmarks/TFT/tft.py)
 - [TabNet based on pytorch (Sercan O. Arik, et al. 2019)](qlib/contrib/model/pytorch_tabnet.py)
+- [DoubleEnsemble based on LightGBM (Chuheng Zhang, et al. 2020)](qlib/contrib/model/double_ensemble.py)
```
[the remark this answers is not in the document]
Comment (meng-ustc) [reply README.md:235]: The results are up to date now.

File: qlib/contrib/model/double_ensemble.py (hunk 1 above)
Comment (meng-ustc) [reply qlib/contrib/model/double_ensemble.py]: OK. The new names are "base_model" and "num_models".

**Comment by you-n-g** [comment]
@meng-ustc 
It looks great.  Thanks for your great efforts  :)

**Pull request merged**